core/modules/views_ui/views_ui.admin.inc

```diff
@@ -3775,86 +3775,87 @@ function views_ui_add_item_form($form, &$form_state) {
-      '#title' => t('Search'),
-    );
-
-    $groups = array('all' => t('- All -'));
-    $form['options']['controls']['group'] = array(
-      '#type' => 'select',
-      '#title' => t('Filter'),
-      '#options' => array(),
-    );
+  $form['options']['controls'] = array(
+    '#theme_wrappers' => array('container'),
+    '#id' => 'views-filterable-options-controls',
+    '#attributes' => array('class' => array('container-inline')),
+  );
+  $form['options']['controls']['options_search'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Search'),
+  );
 
-    $form['options']['name'] = array(
-      '#prefix' => '<div class="views-radio-box form-checkboxes views-filterable-options">',
-      '#suffix' => '</div>',
-      '#tree' => TRUE,
-      '#default_value' => 'all',
-    );
+  $groups = array('all' => t('- All -'));
+  $form['options']['controls']['group'] = array(
+    '#type' => 'select',
+    '#title' => t('Filter'),
+    '#options' => array(),
+  );
 
-    // Group options first to simplify the DOM objects that Views
-    // dependent JS will act upon.
-    $grouped_options = array();
-    foreach ($options as $key => $option) {
-      $group = preg_replace('/[^a-z0-9]/', '-', strtolower($option['group']));
-      $groups[$group] = $option['group'];
-      $grouped_options[$group][$key] = $option;
-      if (!empty($option['aliases']) && is_array($option['aliases'])) {
-        foreach ($option['aliases'] as $id => $alias) {
-          if (empty($alias['base']) || !empty($base_tables[$alias['base']])) {
-            $copy = $option;
-            $copy['group'] = $alias['group'];
-            $copy['title'] = $alias['title'];
-            if (isset($alias['help'])) {
-              $copy['help'] = $alias['help'];
-            }
+  $form['options']['name'] = array(
+    '#prefix' => '<div class="views-radio-box form-checkboxes views-filterable-options">',
+    '#suffix' => '</div>',
+    '#tree' => TRUE,
+    '#default_value' => 'all',
+  );
 
-            $group = preg_replace('/[^a-z0-9]/', '-', strtolower($copy['group']));
-            $groups[$group] = $copy['group'];
-            $grouped_options[$group][$key . '$' . $id] = $copy;
+  // Build the sortable table header.
+  $header = array(
+    'title' => array('data' => t('Title'), 'sort' => 'asc'),
+    'group' => array('data' => t('Category'), 'sort' => 'asc'),
+    'help' => array('data' => t('Description')),
+  );
+  foreach ($fields as $key => $field) {
+    $group = preg_replace('/[^a-z0-9]/', '-', strtolower($field['group']));
+    $groups[$group] = $field['group'];
+    $fields[$key]['#attributes'] = array(
+      'class' => array(
+        'views-filterable-option',
+        $group,
+      ),
+    );
+    $fields[$key]['help'] = "<div class='description'>" . $field['help'] . "</div>";
+    $fields[$key]['title'] = "<div class='title'>" . $field['title'] . "</div>";
+    $fields[$key]['group'] = "<div class='group'>" . $field['group'] . "</div>";
```

Comment:
Classes and other attributes can be added to table rows/cells in a similar, standardized manner as they can be added to the table header, via arrays. We should be doing that here as well if those `<div>`s are only used in order to add the classes and aren't otherwise explicitly necessary.

Reply:
Thanks @klonos, please see if the last commit addresses your concerns:
https://github.com/backdrop/backdrop/pull/4628/commits/6ac48cf6903e24022e5830797f270016e0b31a91

Reply:
Yup, this also looks much better now - no "manual" HTML cruft 👍🏼 

Thanks again.